File: Data Structures & Algorithms/duplicate-integer/submission-2.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        //boolean isTrue = false;
        HashSet<Integer> st = new HashSet<>();

        for(int num : nums){
            if(!st.add(num)){
                return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-2.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        mp = {}
        frequency = 0
        res = []
        for num in nums:
            mp[num]  = 1 + mp.get(num, 0)

        arr = []
        for num, cnt in mp.items():
            arr.append([cnt, num])
        arr.sort()
        
        while len(res) < k:
            res.append(arr.pop()[1])
        return res



        
            
        